index.js

```diff
@@ -0,0 +1,11 @@
+const burgerButton = document.getElementById('burger');
+const closeButton = document.getElementById('close');
+const mobileNav = document.querySelector('.mobile-nav');
+
+burgerButton.addEventListener('click', () => {
+  mobileNav.style.display = 'flex';
```

Comment:
The project requirement comprises sticking to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) (following the template that you chose).

Instead of applying a single style property 'display', I'd advise using a separate class and toggle it here. This would help in styling multiple properties in your mobile navbar.

Please fix the following in your mobile navbar styling once you open it.
- [ ] hamburger menu icon shall disappear;
- [ ] logo, in your case "Aziz Mejri", shall not appear;
- [ ] remove text decoration from anchor links (a tags) and give relevant color;
- [ ] currently flex-elements are in center, align them to the left;
- [ ] add bottom border with appropriate styling;
- [ ] make sure that background color of the mobile navbar covers the mobile screen;

Here is the snapshot from your selected template #2
<img width="311" alt="Screen Shot 2021-07-26 at 01 32 25" src="https://user-images.githubusercontent.com/60043698/126912688-bd1f6651-9e0a-495a-9cdd-52ba1bf62df5.png">

Reply:
- [ ] [REQUIRED] It will be perfect if you create a separate class in your CSS file then toggle it when one condition is reached. For best practice, it is good to separate CSS manipulation and JS.

- [ ] [REQUIRED] Good job by trying to fix some of the requirements asked by the previous TSE but, I suggest that you stick to the design as much as possible (e.g., font, colors, images, tests, margins, layout).
As you can see in this screenshot, font colors, alignment and margins are different from the template:
![1](https://user-images.githubusercontent.com/58552748/126953846-2a458b91-a1f4-47c1-ba54-ffea24c541ee.gif)